scripts/tank_cmd.py

```diff
@@ -573,6 +788,13 @@ def _list_commands(log, tk, ctx):
             by_category[x.category] = []
         by_category[x.category].append(x)
 
+    # Add Login category manually, since they are not commands per-se since they are run before the
+    # engine is initialized.
+
+    by_category.setdefault("Login", []).append(
```

Comment:
If I did so, I would create a file with a command that does nothing, with a comment explaining why that is?